client/flags/flags.go

```diff
@@ -76,6 +76,11 @@ const (
 	// Tendermint logging flags
 	FlagLogLevel  = "log_level"
 	FlagLogFormat = "log_format"
+
+	// QGB related flags
+	// FIXME: can we have these
```

Comment:
It's alright to leave these here? 
I checked also for the other flags, they are defined two times, here and under context also.

Reply:
> they are defined two times, here and under context also

I would assume so. where are the second definitions?

Reply:
https://github.com/SweeXordious/cosmos-sdk/blob/f53f9d610cc71e0b67dcf423fade44c7ae9e4a94/client/context.go#L57

Reply:
@evan-forbes Let me know if this is alright so that I merge.
Thanks

Reply:
ahh I see, we actually probably want to remove those, as they are not global flags. Only the global flags for txs get loaded into that context. When we need access to those flags, we should access via parsing the flags in the definition of the cli command